multimedia/ffmpeg/Makefile

```diff
@@ -401,7 +398,34 @@ FFMPEG_CONFIGURE:= \
 	--disable-vaapi \
 	--disable-vda \
 	--disable-vdpau \
-	--disable-outdevs
+	--disable-outdevs 
+
+
+#selectibly disable optimizations according to arch/cpu type
+ifneq ($(findstring arm,$(CONFIG_ARCH)),)
```

Comment:
Enabling THUMB for ffmpeg - really? Besides potential size savings how badly does it affect performance? Does this option force THUMB or is it optional depending other compiler options?